snap/README.md

```diff
@@ -40,6 +40,24 @@ The App Service Configurable application service allows a variety of use cases t
 
 /var/snap/edgex-app-service-configurable/current/config/res/
 
+### Startup environment variables
+
+The startup behaviour can be modified through environment variables.
```

Comment:
suggest:

EdgeX services by default wait 60s for dependencies (e.g. Core Data) to become available, and will exit after this time if the dependencies aren't met. The following options can be used to override this startup behavior on systems where it takes longer than expected for the dependent services provided by the edgexfoundry snap to start. Note, both options below are specified as a number of seconds.